ui/perfherder/helpers.js

```diff
@@ -392,16 +392,8 @@ export const getGraphsURL = (
   // the otherwise rather useless signature hash to avoid having to fetch this
   // information from the server)
   if (phFrameworksWithRelatedBranches.includes(performanceFrameworkId)) {
-    const branches =
-      alertRepository === 'mozilla-beta'
-        ? ['autoland']
-        : thPerformanceBranches.filter(branch => branch !== alertRepository);
-    url += branches
-      .map(
-        branch =>
-          `&series=${branch},${alert.series_signature.signature_hash},1,${alert.series_signature.framework_id}`,
-      )
-      .join('');
+    const branch = alertRepository === 'mozilla-beta' ? ['autoland'] : [];
```

Comment:
I think it'd be better to keep the original `branches` plural. It's possible that in the future we'd plot multiple related branches, not just one.

I like the empty `[]` approach.